packages/react/src/hooks/useCallbackThrottle/useCallbackThrottle.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+
+import { useCallbackThrottle } from './useCallbackThrottle';
+
+const meta: Meta = {
+  tags: ['autodocs'],
+  title: 'Hooks/useCallbackThrottle',
+  parameters: {
+    references: ['useCallbackThrottle'],
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const Demo: Story = {
+  render: function Render() {
+    const [value, setValue] = useState('');
+    const [throttledValue, setThrottledValue] = useState('');
+
+    useCallbackThrottle(() => {
```

Comment:
Должно быть
```suggestion
    const onChange = useCallbackThrottle(() => {
```
И потом его в инпут прокидываем